pom.xml

```diff
@@ -1446,6 +1446,7 @@
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
         <version>${junit.version}</version>
+        <type>test-jar</type>
```

Comment:
We duplicate the junit definition around line 1400 as well so we should be able to remove this entire block here as well.

Reply:
Same as above.